src/sass/_profile_detail_overview.scss

```diff
@@ -12,6 +12,10 @@
     font-weight: 500;
   }
 
+  .row {
+    margin-top: $spv--x-large;
+  }
+
```

Comment:
```suggestion
```
I might be missing something, but it seems this has no effect as it is overridden and never applied?